### PR TITLE
PWGEM/PhotonMeson: replace aod::Tracks with aod::TracksIU

### PR DESCRIPTION
**Pull request by dsekihat**
replace aod::Tracks with aod::TracksIU
remove name "KF" in cut settings.

### CODE REVIEW

File: PWGEM/PhotonMeson/TableProducer/createEMReducedMCEvent.cxx [hunk 1]
```diff
@@ -136,15 +136,31 @@ struct createEMReducedMCEvent {
...
+        //       LOGF(info,"pi0 from K0S : index = %d , mc track pdg = %d , producedByGenerator =  %d , isPhysicalPrimary = %d", mctrack.index(), mctrack.pdgCode(), mctrack.producedByGenerator(), mctrack.isPhysicalPrimary());
+        //     }
+        //     else if(pdg_mp == 223){
+        //       LOGF(info,"pi0 from omega : index = %d , mc track pdg = %d , producedByGenerator =  %d , isPhysicalPrimary = %d", mctrack.index(), mctrack.pdgCode(), mctrack.producedByGenerator(), mctrack.isPhysicalPrimary());
+        //     }
+
+        //   }
+        // }
+
+        // Note that pi0 from weak decay gives producedByGenerator() = false
```
Comment (mhemmer-cern) [inline PWGEM/PhotonMeson/TableProducer/createEMReducedMCEvent.cxx:156]: why add this code if it is commented out?

**Review by mhemmer-cern — APPROVED** [review]
Looks good

File: PWGEM/PhotonMeson/TableProducer/createEMReducedMCEvent.cxx (hunk 1 above)
Comment (dsekihat) [reply PWGEM/PhotonMeson/TableProducer/createEMReducedMCEvent.cxx:156]: it was only test. I will remove these lines in the next PR.

Comment (mhemmer-cern) [reply PWGEM/PhotonMeson/TableProducer/createEMReducedMCEvent.cxx:156]: Okay :+1:

**Pull request merged**